### PR TITLE
OKTA-601609 Add separate ISV guide note in OIN submission requirements

### PR DESCRIPTION
**Pull request by vanngo-okta**
## Description:
- **What's changed?** Update OIN submission requirements for Customer configuration guide: add separate ISV guide note
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-601609](https://oktainc.atlassian.net/browse/OKTA-601609)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md [hunk 1]
```diff
@@ -132,13 +132,15 @@ The test account allows the OIN team to verify that your integration flow works
 
 A configuration guide helps your customers understand how to configure your Okta integration to work with your cloud application.
 
-Provide a configuration guide as part of the OIN submission process for all integrations:
+Provide a configuration guide as part of the OIN submission process for each type of integration:
```
Comment (ykrestyanov-okta) [inline packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md:135]: @vanngo-okta Can we replace it with:
"Provide a separate configuration guide as part of the OIN submission process for each type of integration"

**Review by ykrestyanov-okta — CHANGES_REQUESTED** [review]

File: packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md (hunk 1 above)
Comment (tovamarkowitz-okta) [reply packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md:135]: Agree with @ykrestyanov-okta.

**Review by tovamarkowitz-okta — APPROVED** [review]

**Review by ykrestyanov-okta — APPROVED** [review]
LGTM

**Review by susanharper-okta — APPROVED** [review]
LGTM

**Pull request merged**